src/attr/_make.py

```diff
@@ -1883,12 +1942,16 @@ class Attribute(object):
     """
     *Read-only* representation of an attribute.
 
-    :attribute name: The name of the attribute.
+    :attribute ``name``: The name of the attribute.
+    :attribute ``inherited``: Whether or not that attribute has been inherited
+       from a base class.
 
     Plus *all* arguments of `attr.ib` (except for ``factory``
     which is only syntactic sugar for ``default=Factory(...)``.
 
-    For the version history of the fields, see `attr.ib`.
+    ..versionadded:: 20.1.0 *inherited*
+
+    For the full version history of the fields, see `attr.ib`.
```

Comment:
```suggestion
    For the full version history of the fields, see ``attr.ib``.
```

Reply:
Same here. All glory to the default role.